Documentation/source/configuration/connections.rst

```diff
@@ -46,3 +46,16 @@ The port where search service is reachable. E.g. default ``9200`` for Elasticsea
 Example::
 
     plugin.tx_searchcore.settings.connections.elasticsearch.port = 9200
+
+.. _index:
+
+``index``
+--------
+
+Used by: :ref:`Elasticsearch`.
+
+The index where the documents are being indexed to. E.g. default ``typo3content`` for Elasticsearch.
```

Comment:
I would add that this is useful to use the same Elasticsearch server for multiple TYPO3 installations. It's not possible to define different indexed for different record types, as this is configured on a connection base.